README.md

```diff
@@ -12,48 +12,73 @@ Python with the scientific stack, compiled to WebAssembly.
-When used inside a browser, Python has full access to the Web APIs.
-
-While closely related to the [iodide project](https://iodide.io), a tool for *literate scientific computing and communication for the web*, Pyodide goes beyond running in a notebook environment. To maximize the flexibility of the modern web, **Pyodide** may
-be used standalone in any context where you want to **run Python inside a web
-browser**.
+Pyodide provides transparent conversion of objects between Javascript and
+Python. When used inside a browser, Python has full access to the Web APIs.
+Pyodide may be used in any context where you want to run Python inside a web
+browser.
 
 ## Try Pyodide (no installation needed)
 
-Try pyodide in [Python REPL](https://pyodide-cdn2.iodide.io/v0.17.0a2/full/console.html) directly in your
-browser.
-
-For further information, look through the [documentation](https://pyodide.org/en/0.17.0a2/).
+Try Pyodide in a
+[REPL](https://pyodide-cdn2.iodide.io/v0.17.0a2/full/console.html) directly in
+your browser. For further information, see the
+[documentation](https://pyodide.org/en/0.17.0a2/).
 
 ## Getting Started
 
-Pyodide offers three different ways to get started depending on your needs and technical resources.
-These include:
+Pyodide offers three different ways to get started depending on your needs and
+technical resources. These include:
 
-- Use hosted distribution of pyodide: see [using pyodide from
-  Javascript](https://pyodide.org/en/0.17.0a2/usage/quickstart.html)
-  documentation.
-- Download a pre-built version from this
-  repository's [releases
-  page](https://github.com/pyodide/pyodide/releases/) and serve its contents with
-  a web server.
+- Use a hosted distribution of Pyodide: see the [Getting
+  Started](https://pyodide.org/en/0.17.0a2/usage/quickstart.html) documentation.
+- Download a version of Pyodide from the [releases
+  page](https://github.com/pyodide/pyodide/releases/) and serve it
+  with a web server.
 - [Build Pyodide from source](https://pyodide.org/en/0.17.0a2/development/building-from-sources.html)
   - Build natively with `make`: primarily for Linux users who want to
     experiment or contribute back to the project.
   - [Use a Docker image](https://pyodide.org/en/0.17.0a2/development/building-from-sources.html#using-docker):
     recommended for Windows and macOS users and for Linux users who prefer a
     Debian-based Docker image with the dependencies already installed.
 
+
+## History
+Pyodide was created in 2018 by [Michael Droettboom](https://github.com/mdboom)
+at Mozilla as part of the [iodide
+project](https://github.com/iodide-project/iodide). Iodide is an experimental
+web-based notebook environment for literate scientific computing and
+communication.
+
+Iodide is no longer maintained. To use Pyodide in an interactive client-side
+notebook, you may also be interested in [Starboard
+Notebook](https://starboard.gg/).
```

Comment:
Maybe rather link to https://pyodide.org/en/latest/project/related-projects.html#notebook-environements-ides-repls to be more neutral and leave the choice  up to user? From what I understand there might be other such systems in development so that list could get longer in the future.